TimoCloud-Universal/src/main/java/cloud/timo/TimoCloud/bungeecord/sockets/BungeeStringHandler.java

```diff
@@ -53,6 +54,13 @@ public void handleMessage(Message message, String originalMessage, Channel chann
             case PROXY_EXECUTE_COMMAND:
                 TimoCloudBungee.getInstance().getProxy().getPluginManager().dispatchCommand(TimoCloudBungee.getInstance().getProxy().getConsole(), (String) data);
                 break;
+            case PROXY_SEND_PLAYER: {
+                Map<String, Object> information = (Map<String, Object>) data;
+                String playerUUID = (String) information.get("playerUUID");
+                String serverObject = (String) information.get("serverObject");
+                TimoCloudBungee.getInstance().getProxy().getPlayer(UUID.fromString(playerUUID)).connect(TimoCloudBungee.getInstance().getProxy().getServerInfo(serverObject));
```

Comment:
What if the player is not online anymore?
We should check if the player exists in order to avoid NullPointerExceptions.